source/includes/_portfolio.md

```diff
@@ -0,0 +1,265 @@
+# Portfolio Data
+
+With the introduction of Portfolio-level endpoints, users associated with a data partnership can now submit portfolio-level data on behalf of the reporting entities and fill in the Real Estate Assessment in full. Mind that portfolio-level APIs do not serve all the functionalities in the Portal, such as aggregation, validation, and submission; meaning that API users should still make use of the Portal interface for access to full features.
```

Comment:
...meaning that API users should still make use of the Portal interface for the end-to-end submission flow.